fastlane/Fastfile

```diff
@@ -450,58 +508,64 @@ UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(PROJECT_ROOT_FOLDER, '.configure-files
 
     return if ENV['BUILDKITE_PULL_REQUEST'].nil?
 
-    install_url = "#{INSTALLABLE_BUILD_DOMAIN}/#{upload_path}"
-    qr_code_url = "https://chart.googleapis.com/chart?chs=500x500&cht=qr&chl=#{CGI.escape(install_url)}&choe=UTF-8"
-    comment_body = "You can test the changes on this Pull Request by <a href='#{install_url}'>downloading an installable build (#{filename})</a>, or scanning this QR code:<br><a href='#{install_url}'><img src='#{qr_code_url}' width='250' height='250' /></a>"
-
-    comment_on_pr(
-      project: GHHELPER_REPO,
-      pr_number: Integer(ENV['BUILDKITE_PULL_REQUEST']),
-      reuse_identifier: "simplenote-installable-build-link",
-      body: comment_body
+    install_url = "#{PROTOTYPE_BUILD_DOMAIN}/#{upload_path}"
+    comment_body = prototype_build_details_comment(
+      app_display_name: 'Simplenote Android',
+      download_url: install_url,
+      metadata: {
+        'Build Type': prototype_build_type
+      }
     )
 
+    unless ENV['BUILDKITE_PULL_REQUEST'].nil?
```

Comment:
💄  It would probably make sense to move the `comment_body = ` lines from above within this `unless` block, as that variable is only useful to compute if we need to call `comment_on_pr`.

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/435f50cdb26e6c104dea6dd5ca8bfdd54c7b5693